MANIFESTO.md

```diff
@@ -1 +1,6 @@
 
+##Community
+
+Mankind did not evolve by sitting alone in the dark. For the software industry to really grow in the Caribbean we need collaboration, knowledge and skill sharing. Avoid platforms and tools that do not respect communities that form around them or worse; shun them.
+
+Community does not only mean software developers, after all we need to solve somebody's problems. Developers should therefore take time to interact with and integrate with the rest of society.
```

Comment:
> Community does not only mean software developers, after all we need to solve somebody's problems.

This came across to me as a comment on products and services and not software in general. This is probably due to 'community' being used as a synonym for the larger society.and 'problems' appearing in the same sentence. 

> There are no software developers on earth who have paying jobs that are not about solving a problem someone else has. I say that without fear of contradiction.

This is debatable but I didn't say that software developers don't solve problems, I was just addressing the context I thought was being communicated.